.github/workflows/build-image.yml

```diff
@@ -0,0 +1,22 @@
+name: Push LSP Docker Images
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  bashls:
+    name: Push bashls to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v2
```

Comment:
`build-push-action` doesn't handle logging into a docker registry any more with v2.

you need to add another step:

```
    name: Login to DockerHub
    uses: docker/login-action@v1
    with:
      username: ${{ secrets.DOCKER_USERNAME }}
      password: ${{ secrets.DOCKER_PASSWORD }}
```

full example (I think):

```
steps:
  - name: Checkout code
    uses: actions/checkout@v2

  - name: Set up Docker Buildx
    uses: docker/setup-buildx-action@v1

  - name: Login to DockerHub
    uses: docker/login-action@v1
    with:
      username: ${{ secrets.DOCKER_USERNAME }}
      password: ${{ secrets.DOCKER_PASSWORD }}

  - name: Build and push
    uses: docker/build-push-action@v2
    with:
      context: subdir/
      tags: myorg/myrepository:latest
```